src/Utilities/SplashHandler.cs

```diff
@@ -0,0 +1,90 @@
+//-----------------------------------------------------------------------------
+// <copyright file="SplashHandler.cs" company="WheelMUD Development Team">
+//   Copyright (c) WheelMUD Development Team.  See LICENSE.txt.  This file is 
+//   subject to the Microsoft Public License.  All other rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------------
+
+using System;
+
+namespace WheelMUD.Utilities
+{
+    public static class SplashHandler
+    {
+        public static string Get(int number = -1)
+        {
+            var random = new Random();
+
+            if (number < 0 || number > SplashScreens.Length - 1)
+            {
+                return SplashScreens[random.Next(0, SplashScreens.Length)];
+            }
+
+            return SplashScreens[number];
+        }
+
+        private static readonly string[] SplashScreens =
+        {
+         "<%green%>" + "<%nl%>" +
+         " _      ____           ____  _____  _____             __  " + "<%nl%>" +
+         "| | /| / / /  ___ ___ / /  |/  / / / / _ \\  ___  ___ / /_" + "<%nl%>" +
+         "| |/ |/ / _ \\/ -_) -_) / /|_/ / /_/ / // / / _ \\/ -_) __/" + "<%nl%>" +
+         "|__/|__/_//_/\\__/\\__/_/_/  /_/\\____/____(_)_//_/\\__/\\__/ " + "<%nl%>" +
```

Comment:
nit: I think we should get rid of the ".net" part;  that was more apt when we were maintaining such a domain instead of going all-in on GitHub and such.